src/components/ChatPanel.tsx

```diff
@@ -75,6 +75,8 @@ export interface ChatPanelProps
    * can click on instead of typing their own.
    */
   messageSuggestions?: string[];
+  /** Link target open behavior on click. */
```

Comment:
nit: can you specify the default behavior in the comment as well?

Reply:
Fixed!